packages/zk-eddsa-event-ticket-pcd/test/ZKEdDSAEventTicketPCD.spec.ts

```diff
@@ -118,7 +122,8 @@ describe("ZKEdDSAEventTicketPCD should work", function () {
     timestampSigned: 1693028498280,
     attendeeSemaphoreId: identity1.getCommitment().toString(),
     isConsumed: false,
-    isRevoked: false
+    isRevoked: false,
+    ticketCategory: TicketCategory.Devconnect
   };
 
```

Comment:
Can you add some test cases below which reveal and examine the ticket-category, to ensure that it's round-tripping properly?  The current test suite actually doesn't include a test case which reveals all fields.  I'd suggest adding one (I already added a fieldsToRevealAll in #633) and ad an expect() to check the value of all ticket fields in the PCD claims.  The verify() step will also implicitly validate those in such a case.

Reply:
Sorry for the confusion - the PR is marked in draft mode. The additional reserved fields + test cases will be added before the PR is marked as ready.